presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestPushAggregationBelowOuterJoin.java

```diff
@@ -256,15 +257,24 @@ public void testPushesAggregationForRightJoin()
                                         tableScan("region", ImmutableMap.of("r_regionkey", "regionkey"))))));
     }
 
-    @Test
-
     public void assertPlan(String sql, PlanMatchPattern pattern)
     {
         List<PlanOptimizer> optimizers = ImmutableList.of(
                 new UnaliasSymbolReferences(),
                 new PruneUnreferencedOutputs(),
                 new PruneIdentityProjections(),
+                new PushAggregationBelowOuterJoin());
+        assertPlanWithOptimizers(sql, pattern, optimizers);
+    }
+
+    public void assertPlanForSubquery(String sql, PlanMatchPattern pattern)
```

Comment:
The predicate push down was making the other tests more complicated, but the subquery won't work without it because that's were the joinCriteria get added.